File: programmers/level 1/descending.js
function descending(n) {
    var answer = "";
    let stringn = String(n);
    let arr = []
    for (var i = 0; i < stringn.length; i++) {
        arr[i] = Number(stringn.charAt(i))
    }
    console.log(arr);
    arr.sort(function(a, b)  {
      return b - a;
    });
    answer = arr.join('');
    return Number(answer);
}

File: programmers/level 1/descending.java
import java.util.*;
import java.io.*;

class descending {
    public long solution(long n) {
        long answer = 0;
        String stringn = Long.toString(n);
        int length = stringn.length();
        String[] arr = stringn.split("");
        
        Arrays.sort(arr, Collections.reverseOrder());
        String temp_answer = String.join("", arr);
        answer = Long.parseLong(temp_answer);
        return answer;
    }
}
